app/models/event.rb

```diff
@@ -362,6 +362,16 @@ def update_rsvp_counts
     )
   end
 
+  def current_state
```

Comment:
the weird thing is... I didn't actually add this. I merged in master prior to doing the PR, but if it was there, then it shouldn't be a diff. I will remove it either way.